test/NuGet.Core.Tests/NuGet.ProjectModel.Test/ProjectLockFile/LockFileUtilitiesTests.cs

```diff
@@ -1127,5 +1127,52 @@ public void IsLockFileStillValid_WithProjectToProjectPackagesConfig_WithIncompat
             Assert.True(actual.IsValid);
             Assert.Empty(actual.InvalidReasons);
         }
+
+        [Fact]
+        public void IsLockFileValid_WithNullVersionRange_DoesNotThrow()
+        {
+            // Arrange
+            var framework = CommonFrameworks.NetStandard20;
+
+            PackageSpec projectA = ProjectTestHelpers.GetPackageSpec(
+                projectName: "A",
+                rootPath: @"C:\",
+                framework: framework.GetShortFolderName(),
+                dependencyName: "PackageA",
+                dependencyVersion: "1.0.0")
+                    .WithTestRestoreMetadata();
+
+            ImmutableArray<LibraryDependency> deps =
```

Comment:
You can create an empty one and then use https://github.com/NuGet/NuGet.Client/blob/27c73ea808bcbead0caea4a387b94f50d14f8c2e/test/TestUtilities/Test.Utility/Commands/ProjectTestHelpers.cs#L328C28-L328C41 to add a dependency if you want. 

It'll shorten the code here.